test/system/src/main/java/io/pravega/test/system/framework/services/MarathonBasedService.java

```diff
@@ -45,7 +45,7 @@
 @Slf4j
 public abstract class MarathonBasedService implements Service {
 
-    static final boolean FORCE_IMAGE = true;
+    static final boolean FORCE_IMAGE = false;
```

Comment:
For my own education, could you elaborate on what this change is doing?

Reply:
This option is not used for mesos container runtime engine. https://mesosphere.github.io/marathon/docs/native-docker.html